compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ClosureDesugar.java

```diff
@@ -972,7 +972,6 @@ public void visit(BLangXMLElementLiteral xmlElementLiteral) {
 
     @Override
     public void visit(BLangXMLTextLiteral xmlTextLiteral) {
-        xmlTextLiteral.textFragments.forEach(this::rewriteExpr);
```

Comment:
What is the reason to not resolving closures of `textFragments` in xmlTextLiteral? Anyway we need to visit the expressions in the `textFragments` and the closure resolving logic will handle in visitors related to each expression.

Reply:
Because `xmlTextLiteral.concatExpr` is made from desugaring `textFragments` and I think `textFragments` are no longer used. Only the `xmlTextLiteral.concatExpr` is used.